frontend/src/index.html

```diff
@@ -21,8 +21,7 @@
       rel="stylesheet"
     />
   </head>
-  <!-- eslint-disable-next-line tailwindcss/no-custom-classname -->
-  <body class="mat-typography">
+  <body>
     <app-root></app-root>
   </body>
 </html>
```

Comment:
There is still a lot to do. Take the version before the update into account. For example, compare the rendered frontend version of this branch to one of our production versions.

You will see many differences, some of them are small, that we need to change.

I'll only list some examples: 

- [ ] The global font size is larger than before.
- [ ] The text on the button has a lot of space between characters.
- [ ] Bold texts are not bold anymore.
- [ ] The "Persistent workspace session" card is wider than before, it takes the whole available space.
- [ ] In the "Menu" dropdown menu, there is a non-clickable right padding (should be moved to the buttons to make it clickable)
- [ ] "Welcome to the Capella Collaboration Manager" is not centered vertically anymore, also the font size is different.

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/20a1625a-2e53-4610-bf01-a8c230f23a43)

The same does also apply to other components.

Reply:
Notes: Buttons have a letter spacing of 1.25 according to [the material desing type system](https://m2.material.io/design/typography/the-type-system.html#type-scale)